heaps/min_heap.py

```diff
@@ -19,21 +18,40 @@ def __init__(self):
     def add(self, key, value = None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: log n
+            Space Complexity: 1
```

Comment:
✨ Because of the recursive call stack of `heap_up` space complexity is O(log n) here